lib/utils.pm

```diff
@@ -168,7 +168,7 @@ Disable screensaver in gnome. To be called from a command prompt, for example an
 
 =cut
 sub turn_off_gnome_screensaver {
-    script_run 'gsettings set org.gnome.desktop.session idle-delay 0';
+    type_string "gsettings set org.gnome.desktop.session idle-delay 0\n";
```

Comment:
How about setting permissions correctly for tty? I guess we have it in consoletest_setup.

Reply:
This will not be a good idea as with the type_string we do not ensure that the command has been executed which already caused some problems. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4768

You could just call `ensure_serialdev_permissions` from lib/utils in your test module "iscsi/iscsi_server". An alternative could be to make sure the `ensure_serialdev_permissions` is called in the parent job already which I think is a better way in the long run.